{"kind":"symbol","identifier":{"interfaceLanguage":"swift","url":"doc:\/\/SwiftErrorKit\/documentation\/SwiftErrorKit\/SourceErrorAttribute\/io"},"sections":[],"hierarchy":{"paths":[["doc:\/\/SwiftErrorKit\/documentation\/SwiftErrorKit","doc:\/\/SwiftErrorKit\/documentation\/SwiftErrorKit\/SourceErrorAttribute"]]},"abstract":[{"text":"Communication error.","type":"text"},{"text":" ","type":"text"},{"type":"text","text":"Network outage, connectivity problem, protocol error, …"}],"variants":[{"paths":["\/documentation\/swifterrorkit\/sourceerrorattribute\/io"],"traits":[{"interfaceLanguage":"swift"}]}],"schemaVersion":{"patch":0,"major":0,"minor":3},"metadata":{"modules":[{"name":"SwiftErrorKit"}],"fragments":[{"kind":"keyword","text":"case"},{"kind":"text","text":" "},{"text":"io","kind":"identifier"}],"role":"symbol","roleHeading":"Case","symbolKind":"case","externalID":"s:13SwiftErrorKit06SourceB9AttributeO2ioyA2CmF","title":"SourceErrorAttribute.io"},"primaryContentSections":[{"kind":"declarations","declarations":[{"languages":["swift"],"tokens":[{"kind":"keyword","text":"case"},{"text":" ","kind":"text"},{"kind":"identifier","text":"io"}],"platforms":["macOS"]}]}],"references":{"doc://SwiftErrorKit/documentation/SwiftErrorKit":{"identifier":"doc:\/\/SwiftErrorKit\/documentation\/SwiftErrorKit","abstract":[{"type":"text","text":"Error attribution and commons."}],"type":"topic","title":"SwiftErrorKit","kind":"symbol","role":"collection","url":"\/documentation\/swifterrorkit"},"doc://SwiftErrorKit/documentation/SwiftErrorKit/SourceErrorAttribute/io":{"role":"symbol","kind":"symbol","type":"topic","title":"SourceErrorAttribute.io","fragments":[{"text":"case","kind":"keyword"},{"text":" ","kind":"text"},{"text":"io","kind":"identifier"}],"url":"\/documentation\/swifterrorkit\/sourceerrorattribute\/io","identifier":"doc:\/\/SwiftErrorKit\/documentation\/SwiftErrorKit\/SourceErrorAttribute\/io","abstract":[{"type":"text","text":"Communication error."},{"text":" ","type":"text"},{"type":"text","text":"Network outage, connectivity problem, protocol error, …"}]},"doc://SwiftErrorKit/documentation/SwiftErrorKit/SourceErrorAttribute":{"identifier":"doc:\/\/SwiftErrorKit\/documentation\/SwiftErrorKit\/SourceErrorAttribute","kind":"symbol","abstract":[{"text":"An error attribute to include information about the high-level component","type":"text"},{"type":"text","text":" "},{"type":"text","text":"in which the error emerged."}],"role":"symbol","title":"SourceErrorAttribute","url":"\/documentation\/swifterrorkit\/sourceerrorattribute","navigatorTitle":[{"kind":"identifier","text":"SourceErrorAttribute"}],"type":"topic","fragments":[{"kind":"keyword","text":"enum"},{"kind":"text","text":" "},{"kind":"identifier","text":"SourceErrorAttribute"}]}}}